### ISSUE TITLE
Fine tune using a pretrained weights

### ISSUE THREAD

**Issue by VanitarNordic** [matterport/Mask_RCNN#237]
Hello,

How can I fine-tune the model on a custom dataset that might have a different number of classes using the pretrained weights?

**Comment by VanitarNordic** [comment]
My data is annotated in the VOC format. Images and masks. Each class has its own color in the masts. 
Background is black and the region between objects and background is white which is known as 'Dontcare'.

In my case there are two classes. Therefore there are some training images and masks with the same file name. One class is background (black) and another class is Red.

**Comment by billybee** [comment]
Hi @VanitarNordic can I ask how do you prepare you own data?  what tool you use to create label and mask in the VOC format?

**Comment by VanitarNordic** [comment]
@billybee 

https://github.com/tzutalin/labelImg

**Comment by VanitarNordic** [comment]
@billybee 

I use a software in the name of `voc-manager` and `VOC2010DatasetManager.exe` for semantic segmentation. I don't remember the website which I downloaded it but if I remember correctly was a inside a university research group. with that you can annotate in the VOC format. search for it in google. if you couldn't find just make a note here for me to try to find it.
